Paper.tex

```diff
@@ -338,7 +337,20 @@ \subsection{The Transaction} \label{subsec:transaction}
 $T_{\mathrm{w}} = 27 + T_{\mathrm{y}}$ or $T_{\mathrm{w}} = 2\beta + 35 + T_{\mathrm{y}}$ (see EIP-155 by \cite{EIP-155}).
 \end{description}
 
-Additionally, a contract creation transaction (regardless whether legacy or EIP-2930) contains:
+There are differences in how one's acceptable gas price is specified in type 2 transactions versus type 0 and type 1 transactions. Type 2 transactions take advantage of the gas market improvements introduced in EIP-1559 and thus have the following fields related to gas price:
+
+\begin{description}
+  \item[maxFeePerGas]\linkdest{max_fee_per_gas}{} A scalar value equal to the maximum number of Wei to be paid per unit of \textit{gas} for all computation costs incurred as a result of the execution of this transaction; formally $T_{\mathrm{m}}$.
```

Comment:
I kept the subscripts for `maxFeePerGas` and `maxPriorityFeePerGas` to a single letter, since the convention of the paper looks to be single letters to denote scalar values within tuples.